.bumpversion.cfg

```diff
@@ -3,15 +3,13 @@ current_version = 0.33.0-alpha
 commit = False
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-[a-z]+)?
-serialize = 
+serialize =
 	{major}.{minor}.{patch}-alpha
```

Comment:
nit: the formatting here is a little weird, would be nice if this was on a single line. Does the gradle formatter do this for some reason?

Reply:
yeah 😑